setup.sh

```diff
@@ -70,10 +70,15 @@ if [ ! -z "${NNS_DAPP_RELEASE:-}" ]
 then
   mkdir -p nns-dapp/out
   curl -L "https://github.com/dfinity/nns-dapp/releases/download/${NNS_DAPP_RELEASE}/nns-dapp.wasm.gz" -o nns-dapp/out/nns-dapp.wasm
-  curl -L "https://github.com/dfinity/nns-dapp/releases/download/${NNS_DAPP_RELEASE}/sns_aggregator_dev.wasm.gz" -o nns-dapp/out/sns_aggregator.wasm
 fi
 
-${DFX} canister install sns_aggregator --network "${NETWORK}" --wasm nns-dapp/out/sns_aggregator.wasm
+if [ ! -z "${SNS_AGGREGATOR_RELEASE:-}" ]
```

Comment:
Why do we need the if statement?

Reply:
There already used to be an if-statement
```
if [ ! -z "${NNS_DAPP_RELEASE:-}" ]
```
covering both the NNS dapp and the SNS aggregator. Since this PR introduces a separate env var for the SNS aggregator, I had to add a new if statement. Now regarding the motivation: the reason is to allow expert people to put their own wasms into nns-dapp/out to install a custom version of the canisters.